src/WebJobs.Script.WebHost/Security/Authentication/Keys/AuthenticationLevelHandler.cs

```diff
@@ -47,13 +49,17 @@ protected override async Task<AuthenticateResult> HandleAuthenticateAsync()
 
             List<ClaimsIdentity> claimsIdentities = new List<ClaimsIdentity>();
 
-            if (_isEasyAuthEnabled)
+            if (_isStaticWebAppsFunction &&
+                Context.Request.TryGetStaticWebAppsIdentity(out ClaimsIdentity staticWebAppsIdentity)
```

Comment:
Ideally we'd just have the Request.GetAppServiceIdentity() call that we had before, and internally that method handles both forms, since both are technically AppService identity.